semantic_conventions.md

```diff
@@ -22,6 +22,9 @@ Span tags apply to **the entire Span**; as such, they apply to the entire timera
 | `db.type` | string | Database type. For any SQL database, `"sql"`. For others, the lower-case database category, e.g. `"cassandra"`, `"hbase"`, or `"redis"`. |
 | `db.user` | string | Username for accessing database. E.g., `"readonly_user"` or `"reporting_user"` |
 | `error` | bool | `true` if and only if the application considers the operation represented by the Span to have failed |
+| `graphql.operation_type` | string | The type of graphql query operation for the associated Span. E.g., `"query"`, `"mutation"`, `"subscription"` |
+| `graphql.operation_name` | string | The name of graphql query operation for the associated Span. E.g., given `query FindFsers { users { name } }` then the operation name is `"FindUsers"` |
+| `graphql.query` | string | The text of graphql query for the associated Span. E.g., `"query"`, `"mutation"`, `"subscription"` |
```

Comment:
I think this actually fits in the scope of `db.query`

Reply:
graphql queries technically have nothing to do with databases.  They are not a proxy for a database query. 

 (As an aside -  some people do implement graphql system via passing through to databases but most do not)

So db.query is not semantically representative of a graphql query.

For example imagine a graphql query like 

```
query {
    orders(key : "1234") {
       orderId
       orderSummary
       billing {
           billingReceiptNumber
           billedName
     }
}
```

the orders "field" might hit the "ordering system" under the covers via HTTP and then hit the "billing system' again via HTTP say to complete this query

Therefore the spans involved will be

```
     span : graphql_server 
             -> span : billing system
             -> span : ordering system
```
So capturing the query text on the span attributes at graphql server entry point is important since it can then be used to work out what was being asked for when it called N downstream systems.

Hence the suggestion for a semantic name here

Reply:
> capturing the query text on the span attributes at graphql server entry point is important

No doubt about that, just wondering if it's that far away from `db.query` to deserve its own prefix here.